### DISCUSSION TITLE
Block interactions!

### DISCUSSION THREAD

**Discussion by yujispin990** [winstxnhdw/lc-hax#99] [Ideas]
Block interactions with anything and/or blocking teleporter commands from another cheater? it sounds like its possible 😅

**Comment by yujispin990** [comment]
or maybe increase cooldown to infinite, probably host only if its possible right?

**Reply by winstxnhdw** [reply]
Yeap

**Comment by winstxnhdw** [comment]
> Block interaction

In some way, `/pumpkin` does that. Try doing `/pumpkin` on yourself. It's absolutely nauseating.

**Reply by yujispin990** [reply]
Ah! but how do i make the pumpkin duration permanent?

**Reply by yujispin990** [reply]
tried 60 and it worked but when i put 120 it became so short 😓

**Reply by winstxnhdw** [reply]
Just set an arbitrarily large duration like `/pumpkin <player> 100000`

**Reply by yujispin990** [reply]
I did, to avoid suspicion, it was so lound and yes nauseating...

**Reply by yujispin990** [reply]
also the pumpkin lags behind a player before finishing the duration to kill it, does it have something to do with my connection or something?

**Reply by yujispin990** [reply]
same with /ct it lags behind the player until it finishes not being able to carry the person up completely and most of the time just stays stuck in the air with the person xd.

**Reply by winstxnhdw** [reply]
Yeah, it's the connection latency.

**Reply by yujispin990** [reply]
Ah! im going to try it later.

**Reply by yujispin990** [reply]
btw, how does the pumpkin actually kill? and does it work with the little plushie dude too?

**Reply by winstxnhdw** [reply]
https://github.com/winstxnhdw/lc-hax/blob/main/lc-hax/Scripts/Commands/PumpkinCommand.cs#L26

It's a lot more straightforward than you think.

**Reply by yujispin990** [reply]
hahahahahaha dude, i thought there was like some sort of pumpkin hitbox collision that deals the same amount of damage as fall damage or something complicated like that XD. It just straight up kills the guy LOL.

**Reply by winstxnhdw** [reply]
`/ct` unintentionally and somehow deals fall damage.

**Reply by yujispin990** [reply]
oh but i noticed when i use /pumpkin on anyone else, it does not kill them though. do they need to stay put or let the pumpkin be on their head until the end of the duration?

**Reply by yujispin990** [reply]
in that case should i just write /pumpkin 1?

**Reply by winstxnhdw** [reply]
The pumpkin only kills when the duration is over.

**Reply by yujispin990** [reply]
sometimes it works, but most of the time the person manages to run away from it or even if he trapped, /ct just stops in mid air and the guy gets stuck.

**Reply by yujispin990** [reply]
yeah thats the thing when i waited for the pumpkin to be over, the dude still lives xd.

**Reply by yujispin990** [reply]
im gonna try with shorter duration.

**Reply by winstxnhdw** [reply]
Either he's in Godmode or `/kill` is still broken.
